ansible_runner/config/_base.py

```diff
@@ -137,7 +137,7 @@ def __init__(self,
 
         os.makedirs(self.artifact_dir, exist_ok=True, mode=0o700)
 
-    _CONTAINER_ENGINES = ('docker', 'podman')
+    _CONTAINER_ENGINES = ('docker', 'podman', 'podman-remote')
```

Comment:
Also note on some systems, there is no `podman-remote` CLI, but instead it is `podman --remote`